sql/core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -36,9 +36,7 @@ trait SharedSQLContext extends SQLTestUtils {
   /**
    * The [[TestSQLContext]] to use for all tests in this suite.
    */
-  protected def ctx: TestSQLContext = _ctx
```

Comment:
I removed the `ctx` alias for `sqlContext`, and changed a lot of lines of code which is trival actually. It's not a big deal to keep an alias, so if you think it's unnecessary, I'll roll back it.